api/auth.go

```diff
@@ -48,8 +40,19 @@ func (a *API) authMiddleware(next http.Handler) http.Handler {
 	})
 }
 
+// postAuthLogin handler to authenticate a user
+//
+//	@Summary		Authenticate a user
+//	@Description	Authenticate a user
```

Comment:
indentation looks wrong, don't think it matters though.

Reply:
The fmt command in the makefile is opinionated. This is correct